modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/menu.js

```diff
@@ -225,6 +225,7 @@ AUI.add(
 						},
 						constrain: true,
 						hideClass: false,
+						modal: Util.isPhone() || Util.isTablet(),
```

Comment:
I added this here because without it, the first time the dropdown is shown in mobile view it doesn't show the gray overlay:

https://user-images.githubusercontent.com/6642927/142867611-2f1f60ba-3811-432f-9247-613dad8b3fc9.mov